### PR TITLE
Split Gull AU and NZ

### PR DESCRIPTION
**Pull request by chuqtas**
Split Gull AU and NZ.
First NSI pull request in some time, hopefully I've done it right...

### CODE REVIEW

File: data/brands/amenity/fuel.json [hunk 1]
```diff
@@ -2199,16 +2199,26 @@
...
         "brand:wikidata": "Q5617739",
         "name": "Gull"
       }
     },
+    {
+      "displayName": "Gull New Zealand",
+      "locationSet": {"include": ["nz"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "Gull New Zealand",
```
Comment (Cj-Malone) [inline data/brands/amenity/fuel.json:2217]: "Gull New Zealand" or "Gull"?

**Comment by UKChris-osm** [comment]
Do they refer to themselves as "_Gull New Zealand_" or just "_Gull_"? If so, it might be better to change:
`"brand": "Gull New Zealand",`
... to be ...
`"brand": "Gull",`

... but retain the `"displayName": "Gull New Zealand",`

**Comment by chuqtas** [comment]
They are just referred to as "Gull" within NZ - refer their website: https://gull.nz/

My reason for the change is: https://community.openstreetmap.org/t/wrong-wikidata-for-gull-in-new-zealand/8999

Happy to go either way regarding @UKChris-osm's comment above!

**Comment by UKChris-osm** [comment]
Thank you for this PR @chuqtas 👍 This should be live in ID within a week.

**Pull request merged**